### PR TITLE
rename segmentrangereader to logreader

### PR DESCRIPTION
**Pull request by jnewb1**

### CODE REVIEW

File: tools/lib/README.md [hunk 1]
```diff
@@ -3,28 +3,19 @@
 Route is a class for conveniently accessing all the [logs](/system/loggerd/) from your routes. The LogReader class reads the non-video logs, i.e. rlog.bz2 and qlog.bz2. There's also a matching FrameReader class for reading the videos.
 
 ```python
-from openpilot.tools.lib.route import Route
```
Comment (adeebshihadeh) [inline tools/lib/README.md:6]: this is still good stuff, i'd just split it out into another section

Comment (jnewb1) [reply tools/lib/README.md]: done
https://github.com/commaai/openpilot/blob/9e054241dbd8b2432972d2c462d37d06a66285a1/tools/lib/README.md#segment-ranges

**Comment by jnewb1** [comment]
@deenlee want to do the same thing for the c++ implementation (for replay and cabana)?

**Comment by sshane** [comment]
that's the wrong @deanlee ^^

**Comment by deanlee** [comment]
@jnewb1 submitted: https://github.com/commaai/openpilot/pull/30981

**Pull request merged**